capsul/engine/local.py

```diff
@@ -8,13 +8,18 @@
 import time
 
 from capsul.api import Pipeline, Process
+from capsul.config import EngineConfiguration
 
 class LocalEngine:
-    def __init__(self, config=None):
+    def __init__(self, label, config=None):
+        self.label = label
         if config is None:
-            self.config = {}
-        else:
+            self.config = EngineConfiguration()
+        elif isinstance(config, EngineConfiguration):
             self.config = config
+        else:
+            self.config = EngineConfiguration()
```

Comment:
engines will only be created by internal code. I am not sure we have to support a dictionary value here.

Reply:
I guess so, I have left it to maintain compatibility with the former config dict, but I guess it's useless.